src/Card.tsx

```diff
@@ -181,17 +188,38 @@ export default function Card({
     fetcher
   );
 
+  if (contact.cardType === undefined) {
+    contact.cardType = "person";
+  }
+  if (contact.location === undefined) {
+    contact.location = "";
+  }
+  
+
+
   const [editMode, setEditMode] = useState(createMode);
+  const [draftCardType, setDraftCardType] = useState("person");
```

Comment:
if this can only be person or repeater, it should be typed as such, e.g.

```ts
useState<"person" | "repeater">("person");
```

Reply:
Oh neat! I was wondering how this got explicitly typed